### PR TITLE
ArxivAPIWrapper - doc_content_chars_max

### LINKED ISSUE #6027: ArxivAPIWrapper
The documentation says:

>     It limits the Document content by doc_content_chars_max.
>     Set doc_content_chars_max=None if you don't want to limit the content size.

But the claim type of int prevents this to be set as None:
https://github.com/hwchase17/langchain/blob/289e9aeb9d122d689d68b2e77236ce3dfcd606a7/langchain/utilities/arxiv.py#LL41C5-L41C38

> ValidationError: 1 validation error for ArxivAPIWrapper
> doc_content_chars_max
>   none is not an allowed value (type=type_error.none.not_allowed)

Can you change that?

In addition, can you also expose this parameter to the `ArxivLoader`?

Thank you!

### PR DESCRIPTION
**Pull request by hp0404**
This PR refactors the ArxivAPIWrapper class making `doc_content_chars_max` parameter optional. Additionally, tests have been added to ensure the functionality of the doc_content_chars_max parameter.

Fixes #6027 (issue)

### CODE REVIEW

**Comment by dean-boi** [comment]
Noobie question perhaps: why has this got a limit at all? Should it not default to type None?

**Comment by hp0404** [comment]
> Noobie question perhaps: why has this got a limit at all? Should it not default to type None?

It seems that @dev2049 introduced the `doc_content_chars_max` parameter with the default 4000 chars value in this [PR](https://github.com/hwchase17/langchain/pull/4538) - I'm not sure why it's 4000 and not some other value (or None), but we can change it too in this PR if needed.

**Comment by dean-boi** [comment]
Hi again, I made some further changes to the arxiv document loader. I'm not sure what the proceedure is though; should I submit a pull request to your branch [hp0404:arxiv-doc-content-chars] or to [hwchase17:master]?

**Comment by hp0404** [comment]
> Hi again, I made some further changes to the arxiv document loader. I'm not sure what the proceedure is though; should I submit a pull request to your branch [hp0404:arxiv-doc-content-chars] or to [hwchase17:master]?

The best practice in this situation would depend on the development workflow and the specific guidelines the project maintainers set- let's wait for them to respond because I'm not sure as well... e.g.,
- you can submit the pull request to the branch "hp0404:arxiv-doc-content-chars"; this PR gets updated with your commits and possibly merged into the master as a single PR.
- or you could create another/alternative PR directly to the hwchase17:master branch, ignoring this one altogether
- or wait for this PR to be merged and propose additional changes in a separate PR to the updated hwchase17:master branch (that will include this PR)
- etc.

**Review by hwchase17 — APPROVED** [review]

**Pull request merged**
